tests/parser/types/test_function_returns.py

```diff
@@ -0,0 +1,13 @@
+def test_function_return_list_in_tuple(get_contract_with_gas_estimation):
+    list_tester_code = """
+@public
+def foo() -> (uint256, uint256[2]):
+    return (0, [0, 1])
```

Comment:
Please add tests for these type of cases as well:

```
def foo() -> ( bytes[10], uint256, uint256[2]) # dynamic mixed with static.
def foo() -> (int128, int128[10], uint256)  # mixed base types
```

Reply:
We will also require test list-in-tuple returns for variables from memory and storage, such as:

```
a: int128[10]
...
return (self.a, 1)
```
and
```
def foo() ...
   a: int128[3] = [1, 2, 3]
   return (a, 1)
```

Reply:
I've added the mixed base types and it works fine it seems.

Although when including bytes[10], i get something as follows : 

``` web3.exceptions.BadFunctionCallOutput: Could not decode contract function call zoo return data b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\nhellohello\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00' for output_types ['bytes', 'uint256', 'uint256[2]']```

I don't understand what's going on here. Shouldn't the output_types match with ['bytes[10],  'uint256', 'uint256[2]']] ?

Reply:
This probably has to do with dynamic offset not being incremented (and might be related to the bottom issue)?
If you don't figure it out push your latest commit and I can take a look at things.

Reply:
Hey man, havent had the time yet. New job... Feel free to pull my branch. But ill try to take a look at it in the next week though

Reply:
Sure no problem, will pick it up ;)